OpenData/secular printed material 1750-1900/Mannin Veen/document.csv

```diff
@@ -1,33 +1,33 @@
 ﻿Manx,English
 MANNIN VEEN.,DEAR ISLE OF MAN
-Yioaree! my t’ou la’al dy ve,Stranger! If you want to be
-"Reillt ayns sauchys, shee, as fea;","Ruled in safety, peace and quiet;"
-"My t’ou shirrey aash dhyt hene, ","If you seek rest for yourself,"
-"Eunys sheelt, dynloght, as glen ; ","Sober delight, sinless, and pure;"
-Ayns shoh vees oo trean as reen : ,Here you will be valiant and tough:
-"Tar, eisht tar, dys Mannin veen ! ","Come, then come, to dear Mann !"
-"My t’ou imlee ayns dy lheill, ","If you are humble in your bearing,"
+Yioaree! my tou la’al dy ve,Stranger! If you want to be
+"Reillt ayns sauchys, shee, as fea;","Ruled in safety, peace and rest;"
+"My tou shirrey aash dhyt hene, ","If you seek ease for yourself,"
+"Eunys sheelt, dyn_loght, as glen; ","Sober delight, innocent, and pure;"

```

Comment:
@maxw0206 This shows as `dynloght` in the image, was the change intentional?

![](https://www.imuseum.im/Olive/APA/IsleofMan/get/image.ashx?kind=block&href=MNH%2F1833%2F10%2F25&id=Ar0080001&ext=.png)


```suggestion
"Eunys sheelt, dynloght, as glen; ","Sober delight, innocent, and pure;"

```

Reply:
Yes, I was using _ indicating editorial space. But I should have explained.